engine/datastore/index.go

```diff
@@ -1,7 +1,7 @@
 package datastore
 
 // IndexExists check whether the given index exists in the datastore
-func (c *Client) IndexExists(tableName, indexName string) (bool, error) {
+func (c *Client) IndexExists(_, _ string) (bool, error) {
```

Comment:
Why can't we remove those params?

Reply:
As I look into the code... We can completely remove this method (the implementation and from `StorageService` interface),
because it is not used anywhere.

Reply:
Done